### DISCUSSION TITLE
Integrating Microsoft Azure TTS Services?

### DISCUSSION THREAD

**Discussion by DigitalJaguar** [EDCD/EDDI#2532] [EDDI Standalone Q&A] [answered]
Is there a way to integrate microsoft azure TTS services into EDDI, instead of just the default windows voices?

**Comment by Tkael** [comment, marked as answer]
Yes, this is a possibility and you're not the first to suggest such a thing (Ref. #2379). It's simply a matter of finding the time to implement it (including implementing an interface for entering credentials). If you have the time and interest, please feel free to [contribute to EDDI's development](https://github.com/EDCD/EDDI/blob/develop/docs/Contributing.md).
